src/components/ui/EditForm/index.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../Button';
+import EditIcon from '../EditIcon';
+import { convertFromPixelsToRem } from '../../../misc';
+import { inputValidation } from '../../../lib/validator';
```

Comment:
Can you use path aliasing here?

Reply:
ok. which means, I will have to add `__` to the validator file name to.

Reply:
resolved